### DISCUSSION TITLE
New member role - Comment

### DISCUSSION THREAD

**Discussion by katycroffbell** [knowledgefutures/pubpub#2105] [Ideas]
### What's your enhancement request?
Create a new member role "Comment" - this person would be allowed to comment on pubs and create reviews, but not directly edit pub drafts.

### What would this enhancement help you accomplish?
Ability to share drafts with people to comment on or review them without fear of them deleting something.

---

### Any attachments or reference images? (Drag them here)
NA

### What's the URL to your PubPub community?
deepseacapacity.pubpub.org

### Who requested this? (If not you)
me

**Comment by gabestein** [comment]
I'm moving this to a discussion both because I think we already address the immediate need, and because I'd like to see some other folks weigh in on the general theme of reviews/comments.

One note here is that the "view" role effectively gives people the ability to comment but not edit on both drafts and releases at any level that I believe should satisfy the immediate request.

We're also working now on a very basic review feature that would allow admins to send potential reviewers a special URL that would  allow them to fill out a comment form (eventually, this could be structured/customized) without even logging in that I suspect would also be a great potential fit for your request.

The initial version will likely just be an additional type of share link that already exists. We're thinking about also adding the ability to explicitly invite people to review via email, with a special link just for them, etc. down the road.

We'd love to hear more ideas here.

**Reply by katycroffbell** [reply]
Ahhh I see. In google docs (which I use all the time),  "view" is literally view-only permission, and "comment" allows someone to comment. You probably don't want to rename pubpub permissions at this point, but the current name is a bit confusing since you can do more than just view.

But yes, View satisfies my current need. Thanks for clarifying.
